guiEditor/src/components/propertyTab/propertyTab.scss

```diff
@@ -320,15 +320,13 @@
     }
 
     .color-picker {
-        height: calc(100% - 8px);
-        margin: 4px;
-        width: calc(100% - 8px);
+        height: 150px;
+        width: 15px;
 
         .color-rect {
-            height: calc(100% - 4px);
-            border: 2px white solid;
+            height: 15px;
+            border: 1px rgb(26, 26, 26) solid;

```

Comment:
sharable color? is it used somewhere else?

Reply:
No, In fact this might need to be black. Let me check!